src/renderer/services/neo.js

```diff
@@ -326,17 +326,24 @@ export default {
       try {
         const inMemory = _.get(store.state.transactionDetails, hash);
         if (inMemory) {
-          inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
-          inMemory.confirmations = inMemory.currentBlockHeight - inMemory.block;
+          if (network.getSelectedNetwork().bestBlock) {
+            inMemory.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
+            inMemory.confirmations = inMemory.currentBlockHeight - inMemory.block;
+          }
           return resolve(inMemory);
         }
 
         return rpcClient.getRawTransaction(hash, 1)
           .then((transaction) => {
-            transaction.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
+            if (network.getSelectedNetwork().bestBlock) {
+              transaction.currentBlockHeight = network.getSelectedNetwork().bestBlock.index;
+            }
             if (transaction.confirmations > 0) {
               transaction.confirmed = true;
-              transaction.block = transaction.currentBlockHeight - transaction.confirmations;
+              if (transaction.currentBlockHeight) {
+                // TODO: switch to looking up the block from the blockhash
+                transaction.block = transaction.currentBlockHeight - transaction.confirmations;
```

Comment:
If this doesn't get set it will probably cause line 331 later to throw an exception. So the TODO has to get addressed before this can be pushed.